Log Viewing/ARKLogTableViewController.m

```diff
@@ -503,28 +499,22 @@ + (NSDateFormatter *)sharedDateFormatter;
 
 #pragma mark - Initialization
 
-- (instancetype)initWithDate:(NSDate *)date;
+- (instancetype)init;
```

Comment:
you won't need this once you mark `init` and `new` as `NS_UNAVAILABLE`. See how `ARKDataArchive` does it.